src/ODB2/Definitions/MO_aircraft_16026_Definition.yaml

```diff
@@ -0,0 +1,99 @@
+---
+#
+# yaml IODA import configuration file for Met Office
+# aircraft report number 16026.
+#
+# This yaml file is designed as input for the ioda-converters
+# python script odbapi2nc.py.
+#
+# Use for Met Office report types:
+# 16026
+# (Report type 16029 contains relative_humidity, but 16026 does not.)
+#
+# Do not use for ECMWF reports.
+#
+
+###########################################################
+##
+## Section 1 -
+## Defines how to retrieve needed variables from the 
+## source ODB2 file
+##
+## There are four ways a variable value might be retrieved:
+##    - column_variables
+##    - vertco_variables
+##    - varno_variables
+##    - varno_vertco_variables
+##
+## These four types are described below.
+###########################################################
+
+# column_variables are read from a specific ODB2 column
+#
+# Format: <key: value> = <variable name: column name>
+#
+# NOTE: analysis_date_time (from andate & antime columns) and
+# date_time (from date & time columns) variables are always read and
+# do not need to be specified here.
+column_variables:
+    station_id: statid
+    station_elevation: stalt
+    longitude: lon
+    latitude: lat
+
+# vertco_variables are read from the vertco_reference_1 column
+# when the vertco_type column has the specified value.
+#
+# Format: <key: value> = <variable name: vertco_type value>
+vertco_variables:
+    air_pressure: 11
+
+# varno_variables are read from the obsvalue, obs_error, and
+# QC columns when the varno column has the specified value.
+#
+# Format: <key: value> = <varno value: variable name>
+varno_variables:
+    2: temperature
```

Comment:
We've been using the CF name "air_temperature" for atmospheric sensible temperature. The website that I've been using (http://cfconventions.org/Data/cf-standard-names/64/build/cf-standard-name-table.html) doesn't list "temperature" as a CF name.

I would like to change this back to "air_temperature" here and in the other YAML files. However, I realize that you might have received a request to change it, and if that's the case we should discuss this with whoever made the request.

Thanks!

Reply:
That was totally an oversight on my part when I created the YAML. Good catch. "temperature has now been changed to "air_temperature" in all the relevant YAML files.